pkg/scanner/local/scan.go

```diff
@@ -151,6 +151,10 @@ func (s Scanner) scanOSPkg(target, osFamily, osName string, pkgs []ftypes.Packag
 }
 
 func (s Scanner) scanLibrary(apps []ftypes.Application, options types.ScanOptions) (report.Results, error) {
+	if len(apps) == 0 {
+		log.Logger.Warn("Trivy is unable to scan this library because no lock files were detected")
```

Comment:
Most images don't have a lock file, so I feel this message can be `info` or `debug`. Also, we're planning to support more than just lock files.

```suggestion
		log.Logger.Info("Trivy skips scanning programming language libraries because no supported file was detected")
```

You can fix that little English phrase.